common/dex-auth/base/k8s_ca_contents.pem

```diff
@@ -0,0 +1,3 @@
+  -----BEGIN CERTIFICATE-----
```

Comment:
Indentation intentional because this content is replaced within a string-based configuration file [here](https://github.com/kubeflow/manifests/pull/195/files#diff-8d6c50839cd5e66a2c31e2b7c18d090bR51).